mkosi/config.py

```diff
@@ -310,6 +323,14 @@ def config_default_mirror(namespace: argparse.Namespace) -> Optional[str]:
     return None
 
 
+def config_default_source_date_epoch(namespace: argparse.Namespace) -> Optional[int]:
+    config_env = getattr(namespace, "environment", [])
```

Comment:
This variable can just be inlined, also no need for the default value if you add default_factory_depends, since that will ensure the variable is set.